include/SDL3/SDL_ttf.h

```diff
@@ -2311,6 +2311,21 @@ extern DECLSPEC int SDLCALL TTF_SetFontDirection(TTF_Font *font, TTF_Direction d
  */
 extern DECLSPEC int SDLCALL TTF_SetFontScriptName(TTF_Font *font, const char *script);
 
+/**
+ * Query whether a font is scalable or not.
+ *
+ * Scalability lets us distinguish between outline and bitmap fonts.
+ *
+ * \param font the font to query
+ *
+ * \returns SDL_TRUE if the font is scalable, SDL_FALSE otherwise.
+ *
+ * \since This function is available since SDL_ttf 2.21.0.
```

Comment:
Since this is only going into SDL_ttf 3, I believe this \since label needs to be revised.